catalog/spatial/stanag4559/spatial-stanag4559-source/src/test/java/org/codice/ddf/spatial/ogc/stanag4559/catalog/source/client/Client.java

```diff
@@ -0,0 +1,429 @@
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.GetRelatedFilesRequest;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.HitCountRequest;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.Library;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.LibraryManager;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.MediaType;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.OrderContents;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.OrderMgr;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.OrderMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.OrderRequest;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.PackageElement;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.PackagingSpec;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.ProductDetails;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.ProductMgr;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.ProductMgrHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.Query;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.SortAttribute;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.SubmitQueryRequest;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.TailoringSpec;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.GIAS.ValidationResults;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.AbsTime;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.AbsTimeHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Coordinate2d;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAG;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAGHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAGListHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Date;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.FileLocation;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameListHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameName;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameValue;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Node;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NodeType;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Rectangle;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Time;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UID.Product;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UID._ProductStub;
+import org.omg.CORBA.IntHolder;
+import org.omg.CORBA.ORB;
+
+public class Client {
+
```

Comment:
I tend to instance a class from the main and then make the class variables, etc non-static.